modules/openapi-generator/src/main/resources/kotlin-server/libraries/jaxrs-spec/api.mustache

```diff
@@ -11,8 +11,6 @@ import io.swagger.annotations.*
 {{/useSwaggerAnnotations}}
 
 import java.io.InputStream
-import java.util.Map
-import java.util.List
```

Comment:
Hi @pmwmedia, first of all, thanks for making this PR.

One questions, instead of removing those imports, shouldn't we replace then with the following imports, as the warning says?
```
import kotlin.collections.Map
import kotlin.collections.List
```

Thanks

Reply:
These imports would be redundant as the fully qualified classes are used in the API template. Using `import kotlin.collections.Map` and `import kotlin.collections.List` could just lead to another warning for unused imports.